filing_history_descriptions.yml

```diff
@@ -56,7 +72,10 @@ description:
     'liquidation-in-administration-change-date-of-dissolution' : "**Notice to Registrar** in respect of date of dissolution"
     'liquidation-in-administration-resignation-of-administrator' : "**Notice of resignation of an administrator**"
     'liquidation-in-administration-vacation-of-office' : "**Notice of vacation of office by administrator**"
-    'liquidation-in-administration-appointment-of-replacement-additional-adminstrator' : "**Notice of appointment** of replacement/additional administrator"
+    'liquidation-in-administration-removal-of-administrator-from-office' : "**Notice of order removing administrator** from office"
+    'liquidation-in-administration-administrator-ceasing-to-be-qualified-to-act' : "**Notice of administrator ceasing to be qualified** to act"
+    'liquidation-in-administration-administrator-deceased' : "**Notice of administrator's death**"
+    'liquidation-in-administration-appointment-of-replacement-additional-administrator' : "**Notice of appointment** of replacement/additional administrator"
```

Comment:
This key hasn't changed, so I don't think it's a breaking change.
This is simply a correction of a typo in this file.